onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -836,14 +836,18 @@ def _where_input_wrangler(
         core_ops.aten_index_put_bool,
     ).skip(
         matcher=lambda sample: not (sample.args[0][0].dtype == torch.bool),
-        reason="this Aten overload only support tensor(bool) as args",
+        reason="this Aten overload only support tensor(bool) as indices",
     ),
     TorchLibOpInfo(
         "index_put",
         core_ops.aten_index_put,
     ).skip(
```

Comment:
Is xfail possible?

Reply:
They may succeed with a newer version of onnxruntime.

Reply:
You may use the `enabled_if` option: https://github.com/microsoft/onnxscript/blob/457e52e10ef4f16e471cebf8e21c1e8fad012219/onnxscript/tests/function_libs/torch_lib/ops_test_data.py#L553